cmd/chartmuseum/main.go

```diff
@@ -22,11 +22,11 @@ import (
 	"os"
 	"strings"
 
-	"github.com/chartmuseum/storage"
 	"helm.sh/chartmuseum/pkg/cache"
 	"helm.sh/chartmuseum/pkg/chartmuseum"
 	"helm.sh/chartmuseum/pkg/config"
 
+	"github.com/chartmuseum/storage"
```

Comment:
Can we put this import back w/ the other chartmuseum ones?

Reply:
Do you mean mixing imports from `helm.sh/chartmuseum` and `github.com/chartmuseum`?